krun.py

```diff
@@ -203,13 +200,41 @@ def get_overall_time_estimate_formatter(self):
     def add_eta_info(self, key, exec_time):
         self.eta_estimates[key].append(exec_time)
 
+    def build_schedule(self, config, current):
```

Comment:
Hmm. Well, you aren't wrong, it's everything from the old results file, which includes the audit etc. This is another case where it might make more sense to have a class?